### ISSUE TITLE
Restart only service process. 

### ISSUE THREAD

**Issue by msangel** [JakeWharton/ProcessPhoenix#24]
I have an application with some service, that have an own process:

> \<service
>             android:name=".service.MyService"
>             android:exported="false"
>             android:process=":serviceprocess" /\>

I need restart that service together with its process. It will be cool, if the library will contain method like:
`ProcessPhoenix.restartServiceProcess(MyService.class)`

**Comment by JakeWharton** [comment]
I'd take a PR for this. Would prefer if it was just
`triggerRebirth(Class<?>)` and it worked for any manifest item that can
declare a process.

**Comment by JakeWharton** [comment]
Also please update the sample to demonstrate this case if someone is going to send a PR.

**Comment by msangel** [comment]
@JakeWharton 
I am no longer android developer.

**Comment by JakeWharton** [comment]
Yep, no worries (and congrats!). The feature request is still valid and I'm speaking to any potential contributor, not you specifically.

**Comment by jobhh** [comment]
> I'd take a PR for this. Would prefer if it was just `triggerRebirth(Class<?>)` and it worked for any manifest item that can declare a process.

> Also please update the sample to demonstrate this case if someone is going to send a PR.

See pull request #69.
It also migrated quite some code to Kotlin, hopefully that's acceptable?
